.github/workflows/ci.yml

```diff
@@ -35,13 +35,17 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: ${{ matrix.python-version }}
+      - uses: actions/cache@v2
```

Comment:
if we want to have this I have nothing against, but let's put it in a different PR